.editorconfig

```diff
@@ -1,15 +1,9 @@
-[*.py]
-indent_style = space
-indent_size = 4
-
-[*.js]
-indent_style = space
-indent_size = 4
-
-[*.css]
-indent_style = space
-indent_size = 4
+root = true
```

Comment:
So basically, it doesn't search in my `~` directory because all configs are here now thanks to the `[*]`. 
Awesomeness!